### PR TITLE
navi 0.14.2 (new formula)

### PR DESCRIPTION
**Pull request by denisidoro**
Related: https://github.com/denisidoro/navi

Previous PRs: https://github.com/Homebrew/homebrew-core/pull/44469 and https://github.com/Homebrew/homebrew-core/pull/44514

### CODE REVIEW

File: Formula/navi.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Navi < Formula
+  desc "Interactive cheatsheet tool for the command-line"
+  homepage "https://github.com/denisidoro/navi"
+  url "https://github.com/denisidoro/navi/archive/v0.14.2.tar.gz"
+  sha256 "1e16b8ff440882a2dd8e73bdb55c2b88724b3f0f9844c602ae9fe74f509d0dfb"
+
+  depends_on "fzf" => :build
```
Comment (dawidd6) [inline Formula/navi.rb]: Isn't `fzf` needed at runtime?

**Comment by dawidd6** [comment]
Please squash all these commits into one titled: `navi 0.14.2 (new formula)`.

File: Formula/navi.rb (hunk 1 above)
Comment (denisidoro) [reply Formula/navi.rb]: Yes! Should it be => :runtime?

Comment (dawidd6) [reply Formula/navi.rb]: Just `depends_on "fzf"`.

**Comment by denisidoro** [comment]
Well, I don't know what happened 🤔 
I force pushed the changes you requested and the PR was automatically merged... I hope that's OK!

**Comment by denisidoro** [comment]
Actually, Github merged it because it was a no-op...

**Pull request closed without merging**